virt/virt-web-console-overview.adoc

```diff
@@ -347,43 +347,50 @@ By default, Red Hat templates are stored in the `openshift` project. You can edi
 |====
 =====
 
-// .*InstanceTypes* tab
-// [%collapsible]
+[id="virtualization-instancetype_{context}"]
+=== InstanceTypes tab
 
-// =====
-// [cols="1,3a", options="header"]
-// |====
-// |Element
-// |Description
+:FeatureName: Creating a virtual machine from an instance type
+include::snippets/technology-preview.adoc[]
 
-// |*Volumes project* list
-// |Select the volume to boot your VM from.
+[%collapsible]
+=====
+[cols="1,3a", options="header"]
+|====
+|Element
+|Description
 
-// |*Add volume*
-// |Upload a new volume or use an existing PersistentVolumeClaim (PVC).
+|*Volumes project*
+|The current namespace.
 
-// |*Filter* field
-// |Filter boot sources by operating system.
+|*Select volume to boot from*
+|Select a volume from the list to boot your VM from.
```

Comment:
The web console overview is an exception, when it comes to abbreviations like VM. Because `VirtualMachine` and other Kubernetes objects occur so frequently in the UI itself, the text should use their names, not abbreviations. 

VM -> VirtualMachine